[Quick Tour] Fixed things found by the docbot

wouterj opened the pull request:
Found some time to get the docbot working again. Before showing it in action, we have to make sure the current docs comply with its standards. That means fixing a lot of too long lines, we have a very strict bot now :)

Besides that, this PR also fixes some serial comma, wrong title case and other wrong usage.

wouterj committed:
Fixed things found by the docbot

javiereguiluz commented on quick_tour/the_controller.rst:
```diff
@@ -309,9 +314,9 @@ from any controller::
-They are useful when you need to set a success message before redirecting the
-user to another page (which will then show the message)::
+You can also store "flash messages" that will auto-delete after the next
+request.  They are useful when you need to set a success message before
```
Minor error: two white spaces after `request.`

javiereguiluz commented on quick_tour/the_controller.rst:
```diff
@@ -332,8 +337,8 @@ And you can display the flash message in the template like this:
+That's all there is to it and I'm not even sure you'll have spent the full
+10 minutes. You were briefly introduced to bundles in the first part and
+all the features you've learned about so far are part of the core framework
+bundle.  But thanks to bundles, everything in Symfony can be extended or
```
Minor error: two white spaces after `bundle.`

javiereguiluz commented on quick_tour/the_view.rst:
```diff
@@ -50,34 +50,41 @@ Below is a minimal template that illustrates a few basics, using two variables
-If the template needs variables to generate its contents, pass them as an array
-using the second optional argument::
+To render a template in Symfony, use the ``render`` method from within a
+controller.  If the template needs variables to generate its contents, pass
```
Minor error: two white spaces after `controller.`

javiereguiluz commented on quick_tour/the_view.rst:
```diff
@@ -269,12 +278,12 @@ extensible way.
-Ready for another 10 minutes with Symfony?
+But I'm getting ahead of myself. First, you need to learn more about the
+controller and that's exactly the topic of the :doc:`next part of this tutorial
+<the_controller>`.  Ready for another 10 minutes with Symfony?
```
Minor error: two white spaces after `<the_controller>.`

javiereguiluz commented:
@WouterJ excellent work! Congrats! I'm really looking forward to see the bot in action.

WouterJ commented:
Thanks for your quick review, @javiereguiluz! I've added this to the docbot reviewers and found a couple more occurences. Fixed them & pushed again :heavy_check_mark:

stof commented:
@WouterJ shouldn't you fix the 2.3 branch first, and then fix 2.7 after merging 2.3 into it ?

WouterJ commented:
@stof thanks for noticing it. I've done the changes against 2.3, but it seems that I choose the wrong branch for the PR.

WouterJ commented:
@javiereguiluz you can try it yourself with a (very on-the-edge) release of the docbot: https://github.com/WouterJ/docbot

WouterJ commented:
I've merged this one now. If anyone spots something wrong, please create a PR fixing it :)
